### DISCUSSION TITLE
Github Education Pack not received

### DISCUSSION THREAD

**Discussion by NikolozKereselidze** [community/community#133397] [GitHub Education] [answered]
### Hi everyone,

Hello,

It has been 4 days since my student status was approved, but I have not yet received my benefits. I understand that the usual wait time is 3 days, so I am concerned as it has been longer than expected. Could you please assist me in resolving this issue?

Thank you.

**Comment by davevad93** [comment, marked as answer]
Hi @NikolozKereselidze ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @NikolozKereselidze was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470

**Reply by NikolozKereselidze** [reply]
> Hi @NikolozKereselidze ,
> 
> It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.
> 
> When the benefits become available, the status label will turn purple and change to:
> 
> * `Congrats! Your request for @NikolozKereselidze was approved and your benefits are now available.`
> 
> For further reference you can look at these discussions: #132790 , #131665 , #132470

Thank you so much!

**Reply by davevad93** [reply]
You're welcome @NikolozKereselidze ! 👍

**Comment by vishalkoc2016** [comment]
Hi everyone,
Hello,

It has been 3 days since my student status was approved, but I have not yet received my benefits.  Could you please assist me in resolving this issue?

Thank you.
